src/main/java/com/juniorstart/juniorstart/controller/EmploymentHistoryController.java

```diff
@@ -0,0 +1,47 @@
+package com.juniorstart.juniorstart.controller;
+
+import com.juniorstart.juniorstart.model.EmploymentHistory;
+import com.juniorstart.juniorstart.security.CurrentUser;
+import com.juniorstart.juniorstart.security.UserPrincipal;
+import com.juniorstart.juniorstart.service.EmploymentHistoryService;
+import org.springframework.http.ResponseEntity;
+import org.springframework.security.access.prepost.PreAuthorize;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.UUID;
+
+@RestController
+@RequestMapping("/user/employmenthistory")
+public class EmploymentHistoryController {
+
+    private final EmploymentHistoryService employmentHistoryService;
+
+    public EmploymentHistoryController(EmploymentHistoryService employmentHistoryService) {
+        this.employmentHistoryService = employmentHistoryService;
+    }
+
+    @PreAuthorize("hasRole('USER')")
+    @GetMapping("")
```

Comment:
Z tego co mi się wydaję to powinno się wyszukiwać po publicId, ale nie się @Ejden wypowie.

Reply:
Zgadza się. Na front przekazywane jest tylko publicId. Na backendzie jednak weryfikowanie czy dany obiekt jest równy drugiego powinno się odbywac głównie po UUID. UUID na froncie nie jest znane.

Reply:
@hosu794  @Ejden  poprawione